book/04-git-server/sections/protocols.asc

```diff
@@ -32,6 +38,12 @@ If you specify `file://`, Git fires up the processes that it normally uses to tr
 The main reason to specify the `file://` prefix is if you want a clean copy of the repository with extraneous references or objects left out – generally after an import from another version-control system or something similar (see <<_git_internals>> for maintenance tasks).
 We'll use the normal path here because doing so is almost always faster.
 
+[NOTE]
+====
+The first syntax above implies using the `--local` option for the `git clone` command;
+see the man page for `git clone` to understand precisely what that means.
+====
+
```

Comment:
I don't think this note is necessary. If the user has need of the `--local` flag, they're probably doing something else that would require a visit to the manpage, so they'll find it on their own.

It might be worth saying that the book isn't intended to be a slightly-more-readable version of the manpages. This is a prose description of how to use Git, which means we have to use some storytelling tools to teach. Making sure to mention every command-line flag that's related to what we're talking about doesn't accomplish the goal of making the user better at Git.

Reply:
Point taken, will remove. On the other hand, I think I will go back at some point and add a note about the possibility of multiple .gitignore files, but simply say they're possible but outside the scope of the book so go read the man page. In a case like that, I think knowing there might be more than one is pretty important.